python/lsst/validate/drp/plot.py

```diff
@@ -279,70 +273,49 @@ def plotAM2(*args, **kwargs):
 def plotAM3(*args, **kwargs):
     return plotAMx(*args, x=3, **kwargs)
 
-def plotAMx(rmsDistMas, annulus, magrange,
-            x=None, level="design",
-            plotbase=""):
+def plotAMx(AMx, plotBase=""):
     """Plot a histogram of the RMS in relative distance between pairs of stars.
 
     Inputs
     ------
-    rmsDistMas : list or numpy.array of float
-        RMS variation of relative distance between stars across a series of visi
-ts.
-    annulus : 2-element list or tuple
-        inner and outer radius of comparison annulus [arcmin]
-    magrange : 2-element list or tuple
-        lower and upper magnitude range
-    level : str
-        One of "minimum", "design", "stretch" indicating the level of the specif
-ication desired.
-    x : int
-        Which of AM1, AM2, AM3.  One of [1,2,3].
-
-    Raises
-    ------
-    ValidateError if `rmsDistMas`
-    ValidateError if `x` isn't in `getAstrometricSpec` values of [1,2,3]
-
-    Notes
-    -----
-    The use of 'annulus' below isn't properly tied to the SRD
-     in the same way that srdSpec.AM1, sprdSpec.AF1, srdSpec.AD1 are
-     because the rmsDistMas has already been calculated for an assumed D.
+    AMx : pipeBase.Struct
+        Must contain:
+        AMx, rmsDistMas, fractionOver, annulus, magRange, x, level,
+        AMx_spec, AFx_spec, ADx_spec
+
+    Outputs
+    -------
+    Plot file prefixed with plotBase.
```

Comment:
This is what the routine does but Outputs is meant to be what the routine returns to the caller.